virt-v2v/cold/WORKSPACE

```diff
@@ -192,7 +192,7 @@ container_pull(
 container_pull(
     name = "ubi9-minimal",
     # 'tag' is also supported, but digest is encouraged for reproducibility.
-    digest = "sha256:e9ea62ea2017705205ba7bc55d20827e06abe4fe071f0793c6cae46edd5855cf",  # 9.1.0-1760
+    digest = "sha256:dc02c6aa8199beb8ed13312d7116a94aa87b5412886bbe358845d3f0626c0f1e",  # 9.2-717
```

Comment:
do we still need to use digest instead the tag? it's the only place we follow digest.

Reply:
Probably no reason to stick to it.